### DISCUSSION TITLE
Técnicas/Dicas pra Negociação de Salário

### DISCUSSION THREAD

**Discussion by felipebernardes** [frontendbr/forum#1122] [Discussão]
Oi pessoal, tudo bem? pensei em termos uma issue pra trocarmos técnicas e dicas de negociação de salário. O que vocês acham? bora? 😄 

Vou começar compartilhando duas que aprendi na ~cagada~ prática:

- se você não estiver desesperadamente procurando trocar de emprego (tendo grana pra pagar os boletos dos próximos meses, se o seu emprego atual estiver OK, etc.), você tem vantagem em poder falar tranquilamente "essa oferta é boa, muito obrigado, mas pra mim esse valor não compensa. eu espero um valor de pelo menos X/benefícios Y e Z".
- já recebi carteirada de "olha, não podemos aumentar esse valor. é a média do mercado de acordo com sua posição" e respondi com "eu entendo, mas não acho que a empresa X seja uma empresa mediana e também não sou um profissional mediano. por que voce acha que faz sentido me oferecer um salário mediano?". o recrutador ficou sem o que responder, pediu pra retornar depois e no dia seguinte recebi uma oferta 20% maior. 🤣 

Se a issue vingar eu posto mais outros aprendizados depois :-)

**Comment by phsantiago** [comment]
É sempre bom ver o Love Mondays e tentar se conhecer ao máximo! Quando você sabe seus pontos fortes você consegue argumentar bem, utilizando eles como diferencial.

**Comment by rodrigooler** [comment]
@felipebernardes quando se está trabalhando é normal receber inúmeras ofertas de trabalho e sou total a favor de funcionários buscarem ou terem curiosidade em procurar outras empresas/cargos/salários. E sempre prestar atenção no quanto estão querendo te pagar, com isto você tera bastantes chances de conseguir uma boa negociação, claro que também depende da situação da empresa. Nem sempre a empresa consegue crescer tão rapidamente.

**Comment by wmartins** [comment]
Li esses artigos aqui uma época e achei muito bons:

https://haseebq.com/my-ten-rules-for-negotiating-a-job-offer/
https://haseebq.com/how-not-to-bomb-your-offer-negotiation/

**Comment by WandersonAlves** [comment]
A segunda dica do @felipebernardes é muito boa mesmo.

Quase sempre funciona kkkkkkkkk
